sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -67,15 +75,21 @@
      * which contains {@link AnalyzeSentimentResultCollection}.
      *
      * @param documents The list of documents to analyze sentiments for.
-     * @param options The {@link TextAnalyticsRequestOptions} request options.
-     *
+     * @param options The additional configurable {@link AnalyzeSentimentOptions options} that may be passed when
+     * analyzing sentiments.
      * @return A mono {@link Response} contains {@link AnalyzeSentimentResultCollection}.
+     *
+     * @throws NullPointerException if {@code documents} is null.
+     * @throws IllegalArgumentException if {@code documents} is empty.
      */
     public Mono<Response<AnalyzeSentimentResultCollection>> analyzeSentimentBatch(
-        Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options) {
+        Iterable<TextDocumentInput> documents, AnalyzeSentimentOptions options) {
```

Comment:
This would be a breaking change, are we ok with that?

Reply:
we are not exploring this class. It is a internal used class.

Reply:
sorry, I missed its an internal class!